### ISSUE TITLE
phone app unreachable

### ISSUE THREAD

**Issue by HiOtto** [starryalley/Unquestionify#2]
Do you have any idea about the info of "PhoneApp Unreachable" while using unquestionify with wechat message notification? And the first time of that case, I even cannot get notifications...

**Comment by starryalley** [comment]
See https://github.com/starryalley/Unquestionify-android/issues/3
I've seen this quite often sometimes and I believe it's garmin's CIQ issues. Each time Garmin updates it's Garmin Connect app it could be different though.

**Comment by HiOtto** [comment]
> See [starryalley/Unquestionify-android#3](https://github.com/starryalley/Unquestionify-android/issues/3)
> I've seen this quite often sometimes and I believe it's garmin's CIQ issues. Each time Garmin updates it's Garmin Connect app it could be different though.

Quite annoying problem, even rebooting the phone or resetting the watch cannot solve it. Seems that can do nothing with this kind of api based development... I'll return this watch then..  
Still thanks for ur reply

**Comment by starryalley** [comment]
That's true. It's so frustrating writing any app on CIQ. A software update on CIQ SDK, Garmin Connect, or the watch itself can break things very easily. That's why I've stopped any development on CIQ. But as a Fenix 6 pro owner I'm still quite happy with it when doing workouts.

**Comment by HiOtto** [comment]
The customs returned my watch because the battery inside.. kind of worn out by this silly purchase experience.... Do you have any idea about the possibility to refresh the firmware of the watch?

**Comment by starryalley** [comment]
You can re-install official firmware but not other region's firmware for the same watch. If it were possible I would never have had written this app for myself.
